mu/app.py

```diff
@@ -170,30 +170,33 @@ def excepthook(*exc_args):
     """
     Log exception and exit cleanly.
     """
-    logging.error("Unrecoverable error", exc_info=(exc_args))
-    try:
-        log_file = base64.standard_b64encode(LOG_FILE.encode("utf-8"))
-        error = base64.standard_b64encode(
-            "".join(traceback.format_exception(*exc_args)).encode("utf-8")
-        )
-        p = platform.uname()
-        params = {
-            "v": __version__,  # version
-            "l": str(i18n.language_code),  # locale
-            "p": base64.standard_b64encode(
-                " ".join([p.system, p.release, p.version, p.machine]).encode(
-                    "utf-8"
-                )
-            ),  # platform
-            "f": log_file,  # location of log file
-            "e": error,  # error message
-        }
-        args = urllib.parse.urlencode(params)
-        webbrowser.open("https://codewith.mu/crash/?" + args)
-    except Exception as e:  # The Alamo of crash handling.
-        logging.error("Failed to report crash", exc_info=e)
-    sys.__excepthook__(*exc_args)
-    sys.exit(1)
+    if exc_args[0] != KeyboardInterrupt:
+        logging.error("Unrecoverable error", exc_info=(exc_args))
+        try:
+            log_file = base64.standard_b64encode(LOG_FILE.encode("utf-8"))
+            error = base64.standard_b64encode(
+                "".join(traceback.format_exception(*exc_args)).encode("utf-8")
+            )
+            p = platform.uname()
+            params = {
+                "v": __version__,  # version
+                "l": str(i18n.language_code),  # locale
+                "p": base64.standard_b64encode(
+                    " ".join([p.system, p.release, p.version, p.machine]).encode(
+                        "utf-8"
+                    )
+                ),  # platform
+                "f": log_file,  # location of log file
+                "e": error,  # error message
+            }
+            args = urllib.parse.urlencode(params)
+            webbrowser.open("https://codewith.mu/crash/?" + args)
+        except Exception as e:  # The Alamo of crash handling.
+            logging.error("Failed to report crash", exc_info=e)
+        sys.__excepthook__(*exc_args)
+        sys.exit(1)
+    else:  # It's harmless, don't sound the alarm.
+        sys.exit(0)
```

Comment:
I wasn't sure if there was anything I needed to do to exit gracefully here, but I'll gladly incorporate it in.

Reply:
Apart from my comment above I think everything else should work fine, any `atexit` will still trigger.